functionary/ui/templates/core/workflow_list.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+{% block content %}
+    <div>
+        <div class="level-right">
+            <button class="button level-item is-link m-5"
+                    hx-target="body"
+                    hx-swap="beforeend"
+                    hx-get="{% url 'ui:workflow-create' %}">
+                <span class="icon is-small">
+                    <i class="fa fa-solid fa-plus"></i>
+                </span>
+                <span>Create</span>
```

Comment:
Added a permission check for showing the button.  I want to create a base class or helper function that takes care of loading all of the user's permissions into the context at some point, but for now I just do it in the list view class.

Also, I had to add some spacing for when the button isn't present, otherwise the table looks too flush up against the top banner.  It feels a little hacky, but I'm not sure of a cleaner way to handle that.